### DISCUSSION TITLE
8月7号已付费升级专业版两年，现在依旧是免费版

### DISCUSSION THREAD

**Discussion by rcok0** [getlantern/lantern-forum-cn#811] [General]
你好，账号sales01@rcrcm.com, 8月7日已提前付费升级到两年专业版，现在依旧不能用，麻烦处理一下，谢谢。
![Uploading 支付.jpg…]()

**Comment by MaxMKMax** [comment]
@rcok0 好的. 我幫您提交問題上去. 但我這裡看不到您的支付單據. 被屏蔽了. 您發單據到我的工作郵箱吧. 謝謝啦. 
mike@getlantern.org

**Comment by rcok0** [comment]
您好，
专业版账户已能使用，多谢，绝对是个良心并用心的团队。
祝好








 

 

&nbsp;
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Tue, Aug 27, 2024 05:38 PM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [getlantern/lantern-forum-cn] 8月7号已付费升级专业版两年，现在依旧是免费版 (Discussion #811)

&nbsp;



 
@rcok0 好的. 我幫您提交問題上去. 但我這裡看不到您的支付單據. 被屏蔽了. 您發單據到我的工作郵箱吧. 謝謝啦.
 ***@***.***
